Formula/zsh-async.rb

```diff
@@ -0,0 +1,28 @@
+class ZshAsync < Formula
+  desc "Perform tasks asynchronously without external tools"
+  homepage "https://github.com/mafredri/zsh-async"
+  url "https://github.com/mafredri/zsh-async/archive/v1.8.5.tar.gz"
+  sha256 "3ba4cbc6f560bf941fe80bee45754317dcc444f5f6114a7ebd40ca04eb20910a"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    cp "async.zsh", "async"
+    zsh_function.install "async"
+    prefix.install Dir["*.zsh"]
+  end
+
+  def caveats
+    <<~EOS
+      To activate async, add the following to your .zshrc:
+        autoload -Uz async && async
+    EOS
+  end
+
+  test do
```

Comment:
I'm not sure what this is supposed to be testing. Why the git command?

Reply:
@SMillerDev Because for some reason the `test.zsh` script fails if not ran in a git repository.